src/node_version.h

```diff
@@ -12,23 +12,18 @@
 #define NODE_STRINGIFY_HELPER(n) #n
 #endif
 
-#if NODE_VERSION_IS_RELEASE
-# ifndef NODE_TAG
+#ifndef NODE_TAG
+# if NODE_VERSION_IS_RELEASE
 #  define NODE_TAG ""
-# endif
-# define NODE_VERSION_STRING  NODE_STRINGIFY(NODE_MAJOR_VERSION) "." \
-                              NODE_STRINGIFY(NODE_MINOR_VERSION) "." \
-                              NODE_STRINGIFY(NODE_PATCH_VERSION)     \
-                              NODE_TAG
-#else
-# ifndef NODE_TAG
+# else
 #  define NODE_TAG "-pre"
 # endif
+#endif
+
 # define NODE_VERSION_STRING  NODE_STRINGIFY(NODE_MAJOR_VERSION) "." \
                               NODE_STRINGIFY(NODE_MINOR_VERSION) "." \
                               NODE_STRINGIFY(NODE_PATCH_VERSION)     \
                               NODE_TAG
```

Comment:
Not 100% sure about this, but might this be missing a dash for custom and nightly DISTTYPE? If so, we should probably prepend the dash in node.gyp, where NODE_TAG is defined.